Source/debug.cpp

```diff
@@ -984,6 +985,55 @@ std::string DebugCmdChangeTRN(const string_view parameter)
 	return out;
 }
 
+std::string DebugCmdOpenAllChests(const string_view /*parameter*/)
+{
+	int countOpened = 0;
+	for (int i = 0; i < MAXOBJECTS; i++) {
+		Object &o = Objects[i];
+		if ((o.IsChest() || o.IsSarcophagus()) && !o.IsOpen()) {
+			OperateObject(*MyPlayer, o);
+			countOpened++;
+		}
+	}
+	return fmt::format("{:d} opened", countOpened);
+}
+
+std::string DebugCmdBreakAllBarrels(const string_view /*parameter*/)
+{
+	int countBroken = 0;
+	for (auto &o : Objects) {
+		if (o.IsBarrel() && !o.IsBroken()) {
+			BreakObject(*MyPlayer, o);
+			countBroken++;
+		}
+	}
+	return fmt::format("{:d} broken", countBroken);
+}
+
+std::string DebugCmdKillAll(const string_view parameter)
+{
+	int countKilled = 0;
+	for (size_t i = 0; i < ActiveMonsterCount; i++) {
+		int mi = ActiveMonsters[i];
+		auto &monster = Monsters[mi];
+		if ((monster.hitPoints >> 6) <= 0)
+			continue;
+		if (monster.position.tile == GolemHoldingCell)
+			continue;
+		M_StartKill(monster, *MyPlayer);
+		countKilled++;
+	}
+	return fmt::format("{:d} killed", countKilled);

```

Comment:
For simple cases of just concatenating some strings and numbers, consider using `StrCat`:

```c++
return StrCat(countKilled, " killed");
```

Reply:
Alright